pkg/execution/state/redis_state/queue.go

```diff
@@ -437,6 +446,8 @@ type queue struct {
 
 	// metrics allows reporting of metrics
 	scope tally.Scope
+	meter metric.Meter
```

Comment:
meters are global singletons, so as long as you initialize them like this, you shouldn't need to assign it.
https://github.com/inngest/monorepo/blob/develop/cmd/all-in-one/executor.go#L79-L90

Reply:
Hmm, what would you prefer this be?

We need some way of creating singletons in this package, though, so I'd prefer to be passing down a meter here.  The queue doesn't have access to any global singletons and should declare what it needs.